amundsen_application/static/js/components/DashboardPage/index.spec.tsx

```diff
@@ -0,0 +1,206 @@
+import * as React from 'react';
+import * as History from 'history';
+
+import { shallow } from 'enzyme';
+
+import { DashboardPage, DashboardPageProps, RouteProps } from './';
+import { getMockRouterProps } from '../../fixtures/mockRouter';
+import AvatarLabel from 'components/common/AvatarLabel';
+import LoadingSpinner from 'components/common/LoadingSpinner';
+import Breadcrumb from 'components/common/Breadcrumb';
+import BookmarkIcon from 'components/common/Bookmark/BookmarkIcon';
+import Flag from 'components/common/Flag';
+import TabsComponent from 'components/common/TabsComponent';
+import ChartList from './ChartList';
+import ImagePreview from './ImagePreview';
+
+import * as Constants from './constants';
+
+import * as ConfigUtils from 'config/config-utils';
+import { dashboardMetadata } from 'fixtures/metadata/dashboard';
+import * as LogUtils from 'utils/logUtils';
+
+import { ResourceType } from 'interfaces';
+
+const MOCK_DISPLAY_NAME = 'displayName';
+const MOCK_ICON_CLASS = 'dashboard-icon';
+
+jest.mock('config/config-utils', () => (
```

Comment:
is it used in test?

Reply:
if so, any reason we put test util in the code?

Reply:
Yes, what I'm using this for is to mock certain values on the application configuration, in order to either:
1. Unit test different cases
2. Mock a specific configuration that permanently applies to all these tests, so that if the configuration logic changes that random tests don't start to fail.